apps-rendering/package.json

```diff
@@ -36,11 +36,11 @@
     "@babel/preset-env": "^7.15.6",
     "@creditkarma/thrift-server-core": "^1.0.4",
     "@emotion/jest": "^11.10.0",
-    "@guardian/apps-rendering-api-models": "^1.1.1",
+    "@guardian/apps-rendering-api-models": "^0.0.0-2022-11-01-SNAPSHOT",
```

Comment:
just adding a note here that we need to remember to release non-snapshot versions of these libraries before merging this! I've accidentally merged snapshot versions before 🙊

Reply:
Just a reminder that dependencies are managed differently since #6339. When you come to update these you may need to merge/rebase `main` and then use `yarn` to lock the changes.

Reply:
Perfect, have done!